generator/generator_test.go

```diff
@@ -156,6 +156,44 @@ func TestCreateCatalogsFromProfileWithBadHref(t *testing.T) {
 	}
 }
 
+func TestSubControlsMapping(t *testing.T) {
+
+	profile := profile.Profile{
+		Imports: []profile.Import{
+			profile.Import{
+				Href: &catalog.Href{
+					URL: func() *url.URL {
+						url, _ := url.Parse("/home/minhaj/go/src/github.com/minhaj10p/cgen/NIST_SP-800-53_rev4_catalog.json")
```

Comment:
Trying to parse absolute url `home/minhaj/go/src/github.com/minhaj10p/...`, wouldn't this fail on other machines ?

Reply:
yes, I feel stupid.
can you suggest something else or should I remove this test then?

Reply:
can we switch it to relative paths ?

Reply:
sure!